src/model.jl

```diff
@@ -6,6 +6,8 @@
 # ===
 
 TAU = 0.0
+η_c = 1.0 # charging efficiency
```

Comment:
Can we not make this an unchangeable global constant? Ideally, this would be a parameter of the dynamic problem.